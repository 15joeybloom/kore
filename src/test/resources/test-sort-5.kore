[]

module TEST-EXCEPTION-1

  sort #a{} []

endmodule []
