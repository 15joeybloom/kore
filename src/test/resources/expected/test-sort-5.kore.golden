Definition
    { definitionAttributes = Attributes []
    , definitionModules =
        [ Module
            { moduleName = ModuleName "TEST-EXCEPTION-1"
            , moduleSentences =
                [ MetaSentence (SentenceSortSentence SentenceSort
                    { sentenceSortName = (Id "#a" AstLocationNone) :: Id Meta
                    , sentenceSortParameters = []
                    , sentenceSortAttributes = Attributes []
                    })
                ]
            , moduleAttributes = Attributes []
            }
        ]
    }